src/App.vue

```diff
@@ -429,6 +544,50 @@ export default {
                 'saveProperties': ['theme', 'sections'],
             };
         },
+
+        validateText(importText) {
+            if (! importText.theme) {
+                return;
+            }
+
+            if(this.themes.indexOf(importText.theme) === -1) {
+                return;
+            }
+
+            if(! importText.sections) {
+                return;
+            }
+
+            let sectionsToImport = [];
+
+            importText.sections.forEach(section => {
+                if(! section.title) {
+                    return;
+                }
+
+                let itemsToImport = [];
+                section.items.forEach((item) => {
+                    if (! item.title && !item.url) {
+                        return;
+                    }
+
+                    itemsToImport.push({
+                       title: item.title,
+                       icon: item.icon || '',
+                       image: item.image || '',
+                       url: item.url,
+                   });
+                });
+
+                sectionsToImport.push({
+                    title: section.title,
+                    items: itemsToImport,
+                });
+            });
```

Comment:
Instead of using forEach and an external array to push items in it, use lodash's map function